docs/how-to/commands.md

```diff
@@ -4,12 +4,14 @@
 
 | Target                      | What it does                                                           |
 | --------------------------- | ---------------------------------------------------------------------- |
-| `bazel run //:docs`         | Builds documentation                                                   |
+| `bazel build //:html`       | Builds documentation in Bazel sandbox                                  |
```

Comment:
We need a better name here, as having docs and html is confusing.

And I would not add a build target until we considered the other DRs. The effort and direction in the last months was to get away from bazel build. Maintaining build and run targets at the same time was enormous effort in the past.

We could add an "docs_experimental_sandbox" target, if we have any usecase for it. I don't want to open a lengthy discussion here in parallel to the DRs. But as it stands today, sandbox has only downsides for the sphinx build.